### DISCUSSION TITLE
Hide dates in pages

### DISCUSSION THREAD

**Discussion by LITUATUI** [hbstack/theme-cards#243] [Q&A] [answered]
Is there an option to hide dates in pages and keep them only for posts?

I want to hide dates in pages like [this](https://pushevs.com/about/).

**Comment by razonyang** [comment]
There isn't, do you want to hide dates and reading time, and taxonomies? Or just the dates?

**Reply by LITUATUI** [reply]
Just the dates.

**Comment by razonyang** [comment, marked as answer]
Available since blog's [v0.26.0](https://github.com/hbstack/blog/releases/tag/v0.26.0), tweak the `meta*` [page parameters](https://hbstack.dev/modules/blog/overview/#page-parameters) to achieve this.

Btw, since Hugo v0.123.0, I'd recommend specifying custom params under the [params](https://gohugo.io/content-management/front-matter/#parameters) key.

**Reply by LITUATUI** [reply]
Awesome, thanks again!
